rfcs/0000-safe-transmute.md

```diff
@@ -2645,6 +2645,37 @@ where
 }
 ```
 
+### Removing the Dependency on Const Generics
```

Comment:
Not sure if the extra explanation on const-generics is needed. The RFC is very long, and very detailed and think some of the information is not strictly necessary for evaluating the RFC.